snuba/cli/devserver.py

```diff
@@ -153,6 +153,45 @@ def devserver(*, bootstrap: bool, workers: bool) -> None:
                 ],
             ),
         ]
+        if settings.ENABLE_METRICS_SUBSCRIPTIONS:
+            daemons += [
+                (
+                    "subscriptions-consumer-metrics-counters",
+                    [
+                        "snuba",
+                        "subscriptions",
+                        "--auto-offset-reset=latest",
+                        "--log-level=debug",
+                        "--max-batch-size=1",
+                        "--consumer-group=snuba-metrics-subscriptions-consumers",
+                        "--dataset=metrics",
+                        "--entity=metrics_counters",
+                        "--commit-log-topic=snuba-metrics-commit-log",
+                        "--commit-log-group=metrics_group",
+                        "--delay-seconds=1",
+                        "--schedule-ttl=10",
+                        "--max-query-workers=1",
+                    ],
+                ),
+                (
+                    "subscriptions-consumer-metrics-sets",
+                    [
+                        "snuba",
+                        "subscriptions",
+                        "--auto-offset-reset=latest",
+                        "--log-level=debug",
+                        "--max-batch-size=1",
+                        "--consumer-group=snuba-metrics-subscriptions-consumers",
+                        "--dataset=metrics",
+                        "--entity=metrics_sets",
+                        "--commit-log-topic=snuba-metrics-commit-log",
+                        "--commit-log-group=metrics_group",
+                        "--delay-seconds=1",
+                        "--schedule-ttl=10",
+                        "--max-query-workers=1",
+                    ],
+                ),
+            ]
```

Comment:
What's the relationship between these 2 consumers? They are both consuming from the same topic, same consumer group, same commit log topic, same commit log group. Isn't that going to cause issues? Should this be run as a single multistorage consumer instead?